api/tests/integration/infrastructure/repositories/participant-results-shared-repository_test.js

```diff
@@ -0,0 +1,121 @@
+const { expect, databaseBuilder, mockLearningContent } = require('../../../test-helper');
+
+const participantResultsSharedRepository = require('../../../../lib/infrastructure/repositories/participant-results-shared-repository');
+const Campaign = require('../../../../lib/domain/models/Campaign');
+
+describe('Integration | Repository | Campaign Participation Aggregate Repository', () => {
+
+  describe('#get', () => {
+    it('computes the participant results the complete referential', async () => {
```

Comment:
`computes the participant results for the complete learning content`

Reply:
done